### ISSUE TITLE
some bot names can fail

### ISSUE THREAD

**Issue by bmorel** [Unvanquished/Unvanquished#1813]
[default.cfg.log](https://github.com/Unvanquished/Unvanquished/files/8791359/default.cfg.log) is the file I have used to get that result (renamed as .log because GH).
I have only observed that with human names, probably some unicode stuff around. Need to do more checking.

Logs have for only hint: `AdminExec: ok: -1 "console^*" "console^*" [0] (): bot`

**Comment by necessarily-equal** [comment]
Yeah, so rewording because I totally didn't get what you meant before I saw it in real life,

Sometimes bot don't get assigned a name despite having a list of names long enough, and they just get called Bot#1.

![Screenshot from 2022-05-28 18-40-10](https://user-images.githubusercontent.com/59283660/170834699-6b7e8cfc-68de-489b-ad01-615eef2c2563.png)

**Comment by necessarily-equal** [comment]
This looks like it may be fixed by https://github.com/Unvanquished/Unvanquished/pull/1818

**Comment by Sswweet** [comment]
It seems the longest allowed bot name is 32 ASCII characters. If a name is longer, it gets replaced by this.

**Comment by necessarily-equal** [comment]
I think we did determine that this is not a bug. Maybe we can have a warning telling the name isn't appropriate but that's about it I'd say?

**Comment by bmorel** [comment]
I consider it a bug that the names are silently ignored.
